cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -897,8 +897,7 @@ pub enum TomlEditError {
     /// The `[install]` table was missing entirely
     #[error("'install' table not found")]
     MissingInstallTable,
-    /// Tried to uninstall a package that wasn't installed
-    #[error("couldn't uninstall '{0}', wasn't previously installed")]
+    #[error("couldn't find package '{0}' in the manifest")]
```

Comment:
nb: Since the _item_ to remove could be a package or an install id, would it be better to simplify this message as `couldn't find '{0}' in the manifest` ?

Reply:
I'd say yes

Reply:
Good point, the term "package" can be a bit ambiguous here.

In fact, now it's back to only being the install id, as we extract them in the Core Environment before calling the remove function. Initially, it was designed to handle both package and install id, but after feedback, I moved the handling to the Core Environment. I've updated the wording in the code and logs to "install id" to make it clearer.